model/src/main/kotlin/ScanRecord.kt

```diff
@@ -40,9 +41,10 @@ data class ScanRecord(
         val scanResults: SortedSet<ScanResultContainer>,
 
         /**
-         * The [CacheStatistics] for the scan results cache.
+         * The [StorageStatistics] for the scan results storage.
```

Comment:
Maybe: AccessStatistics?

Reply:
Yeah, I like that idea to make this class name independent of *where* the data is stored. This still *could* be used by some real / other cache implementation to come, after all.

Reply:
Done.